specification/columns/commitmentdiscountpurchasequantity.md

```diff
@@ -0,0 +1,35 @@
+# Commitment Discount Purchase Quantity
+
+Commitment Discount Purchase Quantity is the numeric value of [CommitmentDiscountUnits](#commitmentdiscountunit) that is eligible for amortization for any full or partial upfront purchase of a [*commitment-based discount*](#glossary:commitment-based-discount).
```

Comment:
I'm concerned that the FOCUS dataset provides information about Commitment Discount Purchase Quantity for only 2 out of 3 payment models (available for All Upfront and Partial Upfront, but not for No Upfront).

Reply:
I think this is a valid point.

1. An [issue](https://github.com/FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec/issues/517) to triage was added around the point of `No Upfront` purchases have 0 cost being included as $0 line items in FOCUS when there are significant values in other measure columns.
 
2. I think this is a valid point and something that [Sample Data - Option 1](https://docs.google.com/spreadsheets/d/1AYDPZ4rl90PPEbyQJUaGCUSwZ4s_sfy-8sO2KyNq0aM/edit?pli=1&gid=0#gid=0) better handles than the current Option 2.

Reply:
With Recurring purchases now reintroduced, resolving this comment